lms/djangoapps/instructor_task/models.py

```diff
@@ -38,6 +38,7 @@
 # define custom states used by InstructorTask
 QUEUING = 'QUEUING'
 PROGRESS = 'PROGRESS'
+TASK_INPUT_LENGTH = 1024
```

Comment:
Sorry, I know I said I didn't care how much this was, but I just saw a PR the other day that might want to use it for something that edges on this limit, so maybe 10,000?

Thank you.

Reply:
Thanks for your input @ormsbee. Going to change it to 10,000.